Packages/Newtonsoft.Json-for-Unity.Converters/Editor/UnityConvertersConfigEditor.cs

```diff
@@ -0,0 +1,297 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json.UnityConverters.Configuration;
+using UnityEditor;
+using UnityEditor.AnimatedValues;
+using UnityEngine;
+
+namespace Newtonsoft.Json.UnityConverters.Editor
+{
+    using Editor = UnityEditor.Editor;
+
+    [CustomEditor(typeof(UnityConvertersConfig))]
+    public class UnityConvertersConfigEditor : Editor
+    {
+        private SerializedProperty _useUnityContractResolver;
+        private SerializedProperty _useAllOutsideConverters;
+        private SerializedProperty _outsideConverters;
+        private SerializedProperty _useAllUnityConverters;
+        private SerializedProperty _unityConverters;
+        private SerializedProperty _useAllJsonNetConverters;
+        private SerializedProperty _jsonNetConverters;
+
+        private IList<Type> _outsideConverterTypes;
+        private IList<Type> _unityConverterTypes;
+        private IList<Type> _jsonNetConverterTypes;
+                
+        private AnimBool _outsideConvertersShow;
+        private AnimBool _unityConvertersShow;
+        private AnimBool _jsonNetConvertersShow;
+
+        private readonly Dictionary<string, Type> _converterTypeByName
+            = new Dictionary<string, Type>();
+
+        private GUIStyle _headerStyle;
+        private GUIStyle _boldHeaderStyle;
+
+        private bool _isDirty;
```

Comment:
Codacy found an issue: [Remove the field '_isDirty' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5865055)